src/sql/azdata.d.ts

```diff
@@ -3088,7 +3093,6 @@ declare module 'azdata' {
 		ariaRowCount?: number;
 		ariaColumnCount?: number;
 		ariaRole?: string;
-		focused?: boolean;
```

Comment:
@Benjin You added this property in https://github.com/microsoft/azuredatastudio/commit/faf6eae535aa7d054731fa877f86aa66e1e25426#diff-b77f32af81d98c5f57caa4f8e1db5565 but I didn't see it being used anywhere. Do you know if some other extension was using it?

Reply:
Correct, it's okay to remove, afair.  I added it as part of an attempt to fix something else, but ended up solving the issue in a different way.  I left this around because I figured it'd be useful at some point (we were in the middle of fixing a bunch of focus-related bugs).